kotlin/workflow-ui-android/src/main/java/com/squareup/workflow/ui/WorkflowRunnerViewModel.kt

```diff
@@ -65,28 +68,20 @@ internal class WorkflowRunnerViewModel<OutputT : Any>(
 
   private val subs = CompositeDisposable()
 
-  @Suppress("EXPERIMENTAL_API_USAGE")
-  private val updates =
-    workflowUpdates.updates.asObservable(Dispatchers.Unconfined)
-        .doOnNext { lastSnapshot = it.snapshot }
-        .replay(1)
-        .autoConnect(1) { subs.add(it) }
+  init {
+    subs.add(workflowHost.renderingsAndSnapshots
```

Comment:
Sketchy to do this in the init block, but I'm pretty sure we can do better.

Reply:
See the latest commit.

Reply:
But we could also give it an explicit start method, same thing.